src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -1224,6 +1227,15 @@ public synchronized ResultIterator iterator(String query, int fetchSize, int rea
      */
     protected abstract ResultIterator createResultIterator(Statement statement, String sql) throws DatabaseEngineException;
 
+    /**
+     * Creates a specific {@link ResultIterator} for the engine in place given given prepared statement.
```

Comment:
given * 2

Reply:
:smile: 
This method (incl. javadoc) was not modified, just moved from below to be close to the other `createResultIterator`

but nice catch!